src/main/resources/templates/pebt/confirmBlankEmail.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{confirm-blank-email.title})}"></head>
+<body>
+<div class="page-wrapper">
+    <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+    <section class="slab">
+        <div class="grid">
+            <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+            <th:block th:if="${true}">
+                <main
+
+                        id="content" role="main" class="form-card spacing-above-35">
+                    <div class="form-card__content">
+                        <th:block th:replace="~{'fragments/icons' :: contactInfo}"></th:block>
+                        <h1 class="h2"
+                            th:text="#{confirm-blank-email.title}"></h1>
+                        <p th:text="#{confirm-blank-email.subtext}"></p>
+                    </div>
+                    <div class="form-card__footer">
+                        <a
+                                th:href="'/flow/pebt/contactInfo'"
+                                th:text="#{confirm-blank-email.add-email}"
+                                class="button button--primary"></a>
+                        <a
+                                th:href="'/flow/pebt/confirmBlankEmail/navigation'"
+                                th:text="#{confirm-blank-email.continue-without}"
+                                class="button button--other"
```

Comment:
Shame we don't have better fragments for these - but I think I see why you weren't able to use `continueButton` here.

Reply:
BTW do you about this `/navigation` feature of form-flow-library? It's neat. It's a server-side HTTP endpoint that redirects to wherever `next` is, dynamically computed based on the flow. It's what continue button uses, which how I figured that out.